examples/gno.land/p/demo/stack/stack.gno

```diff
@@ -0,0 +1,43 @@
+package stack
+
+type Stack struct {
+	top    *node
+	length int
+}
+
+type node struct {
+	value interface{}
```

Comment:
Just curious about this - do we support generics for `.gno`?

You could easily turn this into:
```go
type node[T any] struct {
    value T
    prev *node
}
```

This way, we don't have to do the clunky type conversion each time you pop on the caller side

Reply:
I don't think we support generics yet.